src/battle_script_commands.c

```diff
@@ -815,7 +815,7 @@ void (* const gBattleScriptingCommandsTable[])(void) =
     Cmd_trysetfutureattack,                      //0xC3
     Cmd_trydobeatup,                             //0xC4
     Cmd_setsemiinvulnerablebit,                  //0xC5
-    Cmd_jumpifweathercheckchargeeffects,         //0xC6
+    Cmd_tryfiretwoturnmovenowcheckeffects,       //0xC6
```

Comment:
misalignment

Reply:
I don't see any misalignment?

Reply:
Am I blind again?
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/93446519/40d4e32f-25e4-4bb6-afc7-f3ecb01729d7)

Reply:
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/16993385/d8a74d3a-1b61-41fc-9e4c-06f5330ab5b5)
It also looks fine on my end, weirdly enough

Reply:
It's time to throw away firefox

Reply:
I'm on Firefox too, haha

Reply:
I don't know then lol